src/dataspec/base.py

```diff
@@ -590,7 +590,28 @@ def _enrich_errors(
         yield error.with_details(tag, loc=loc)
 
 
-def compose_conformers(
+def compose_conformers(*conformers: Conformer) -> Conformer:
+    """
+    Return a single conformer which is the composition of the input conformers.
+
+    If a single conformer is given, return the conformer.
+    """
+
+    if len(conformers) == 1:
+        return conformers[0]
+
+    def do_conform(v):
+        conformed_v = v
+        for conform in conformers:
+            conformed_v = conform(conformed_v)
+            if conformed_v is INVALID:
+                break
```

Comment:
This block is repeated in `compose_spec_conformers`. An opportunity for reuse?

Reply:
I decided not to, because that's using `Spec` instances and this is dealing directly with `Conformer`s.